src/commands/GameIntegration/brawlstars.ts

```diff
@@ -61,7 +62,7 @@ export interface BrawlStarsGIData {
 		'tag',
 		(arg, _possible, message) => {
 			if (kTagRegex.test(arg)) return arg;
-			throw message.language.get('commandClashofclansInvalidPlayerTag', { playertag: arg });
+			throw message.language.get(LanguageKeys.Commands.GameIntegration.ClashofclansInvalidPlayerTag, { playertag: arg });
```

Comment:
This should be a separate key for BrawlStars, since this one mentions Clash of Clans specifically.

Reply:
Fixed in 16870cd4.